Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,50 @@
+# Microcontroller Signoff
+## FUNCTION OF THE SUBSYSTEM
+The microcontroller and its code are for operating, managing, communicating with, maintaining, and collaborating with all of the other subsystems and acts as both the brain and body of the device.  All the data and instructions must flow through the microcontroller giving a common place for all of the data to go and be readily available for whichever subsystem has need of it.  The hardware will be responsible for the actual actions and completion of any given task.  The code will be responsible for controlling the hardware and manipulating data.  This subsystem will be where most of the coding for the project is done as the operating code will be the largest by far.
+
+These jobs require a sufficiently powerful board.  One that has ample storage, plenty of physical space, decent processing power, sufficient support, and none restrictive.  Before choosing components and detailing the design, the specific constraints of the subsystem need to be known and laid out.  Then any potential solutions can be compared to the constraints to test viability and fulfillment of the subsystem’s purpose. 
+
+## CONSTRAINTS ON THE SUBSYSTEM
+The constraints given below are generalized to allow for comparison of any potential contender for the microcontroller.  Constraints on the subsystem are as follows:
+1. The chosen microcontroller should cost between $20-$60.  This range is gathered from estimating the costs of other components and viewing prices of avaiable options that meet the needs of the subsystem.
+2. The microcontroller must support communication between components.  To do this, Serial Peripheral Interface (SPI), Universal Asynchronous Receiver/Transmitter (UART) as a standard in hardware communication for liaising will need to be achievable on the board. 
+3. External storage support is required as per the shall statements. Here, the most preferred standard for external storage is a Universal Serial Bus (USB) Flash Drive as it is common in households and easily available.
```

Comment:
Why? You can't arbitrarily claim that USB is best.

Reply:
That was not our intention.  Explaining why it was chosen.